<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clipPlanesController</class>
 <widget class="QWidget" name="clipPlanesController">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>239</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Clip Planes</string>
  </property>
  <widget class="QWidget" name="clipPlanesWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>251</width>
     <height>211</height>
    </rect>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>251</width>
      <height>176</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="clipPlanesLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="posXLabel">
       <property name="text">
        <string>+X</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="posXSlider">
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>10000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="negXLabel">
       <property name="text">
        <string>-X</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="negXSlider">
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="posYLabel">
       <property name="text">
        <string>+Y</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSlider" name="posYSlider">
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>10000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="negYLabel">
       <property name="text">
        <string>-Y</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="negYSlider">
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="posZLabel">
       <property name="text">
        <string>+Z</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSlider" name="posZSlider">
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>10000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="negZLabel">
       <property name="text">
        <string>-Z</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSlider" name="negZSlider">
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QDoubleSpinBox" name="posXSpinBox">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="minimum">
        <double>-0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000010000000000</double>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QDoubleSpinBox" name="negXSpinBox">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="minimum">
        <double>-0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000010000000000</double>
       </property>
       <property name="value">
        <double>-0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDoubleSpinBox" name="posYSpinBox">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="minimum">
        <double>-0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000010000000000</double>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QDoubleSpinBox" name="negYSpinBox">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="minimum">
        <double>-0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000010000000000</double>
       </property>
       <property name="value">
        <double>-0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QDoubleSpinBox" name="posZSpinBox">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="minimum">
        <double>-0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000010000000000</double>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QDoubleSpinBox" name="negZSpinBox">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="decimals">
        <number>4</number>
       </property>
       <property name="minimum">
        <double>-0.500000000000000</double>
       </property>
       <property name="maximum">
        <double>0.500000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.000010000000000</double>
       </property>
       <property name="value">
        <double>-0.500000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="clipPlanesResetButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>4</y>
      <width>120</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="text">
     <string>Reset Clip Planes</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>130</width>
     <height>17</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="clipPlanesLabel">
      <property name="text">
       <string>Enable Clip Planes</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="clipPlanesCheckBox">
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
